extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -19,15 +19,21 @@ import java.util.*
  * `["format",["coalesce",["get","name_en"],["get","name"]],{}]` will be replaced by
  * `["format",["coalesce",["get","name_zh-Hans"],["get","name"]],{}]`
  */
-internal fun setMapLanguage(locale: Locale, style: StyleInterface) {
+internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: List<String>?) {
   style.styleSources
     .filter { sourceIsFromMapbox(style, it) }
     .forEach { source ->
       style.styleLayers
         .filter { it.type == LAYER_TYPE_SYMBOL }
+        .filter { layer ->
```

Comment:
Could concat with previous filter with `&&`

Reply:
Yeah, I have thought in this way too, but with another filter, both the code and logic are cleaner.